meson.build

```diff
@@ -286,20 +287,20 @@ if not get_option('tensorrt-support').disabled()
     cuda_dep = dependency('cuda-' + ver, required: false)
     cudart_dep = dependency('cudart-' + ver, required: false)
     if cuda_dep.found() and cudart_dep.found()
-      if ver != '11.0' and ver != '10.2'
+      if ver != '12.1' and ver != '11.0' and ver != '10.2'
         message('Warning: the recommended cuda version is at least 10.2')
       endif
       break
     endif
   endforeach
 
   nvinfer_lib = cxx.find_library('nvinfer', required: false)
-  if nvinfer_lib.found() and cxx.check_header('NvInfer.h')
+  if nvinfer_lib.found() #and cxx.check_header('NvInfer.h')
     nvinfer_dep = declare_dependency(dependencies: nvinfer_lib)
   endif
 
   nvparsers_lib = cxx.find_library('nvparsers', required: false)
-  if nvparsers_lib.found() and cxx.check_header('NvUffParser.h')
+  if nvparsers_lib.found() #and cxx.check_header('NvUffParser.h')
```

Comment:
Please do not comment it out. Remove or leave it.

Reply:
Done